### PR TITLE
Add Zone Management Section to Monoprice docs

### PR DESCRIPTION
**Pull request by OnFreund**
## Proposed change

A new section about Zone management (which changed significantly with the move to config entries). This was an inadvertent omission on my part when I initially updated the docs.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/30337
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by OnFreund** [comment]
I also believe this should be cherry picked to the RC branch.

File: source/_integrations/monoprice.markdown [hunk 1]
```diff
@@ -22,6 +22,11 @@ Select the serial port to which Monoprice amplifier is connected, and name the s
 
 Sources can also be later edited from the integration options (gear icon in the top right when selecting the integration). In order to remove an existing source, you'll need to replace it with a space (simply removing its name will still keep it in place). Note that editing sources will remove the snapshot you may have saved.
 
+## Zone Management
+
+Devices and entities are created for each of the possible 18 zones, and can be enabled, disabled, and renamed through regular Home Assisant methods.
+By default, the first 6 zones (11..16) are enabled, while the other 12 zones possible through extensions (21..26, and 31..36) are disabled.
+
```
Comment (frenck) [inline source/_integrations/monoprice.markdown]:
```suggestion
Devices and entities are created for each of the possible 18 zones, and can be enabled, disabled and renamed through regular Home Assistant methods.
By default, the first 6 zones (11..16) are enabled, while the other 12 zones, possible through extensions (21..26, and 31..36), are disabled.

```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @OnFreund!
Small review suggestion, could you take a look? Thanks!

File: source/_integrations/monoprice.markdown (hunk 1 above)
Comment (OnFreund) [reply source/_integrations/monoprice.markdown]: Thanks @frenck.
The first change is a matter of convention (Oxford comma), so I'll go with whatever the convention in the docs is.
However, the additional commas in the second sentence make it less readable, I believe. "while the other 12 zones possible through extensions" is a single phrase, and the zone numbers also belong in it.
A 3rd alternative would be "while the other 12 zones (21..26 and 31..36), which are possible through extensions, are disabled", but I think it's more cumbersome.

Comment (frenck) [reply source/_integrations/monoprice.markdown]: We don't do Oxford comma's, see: <https://developers.home-assistant.io/docs/documentation_standards#general-documentation>

As for the other one, that is optional. 🤷‍♂

Comment (OnFreund) [reply source/_integrations/monoprice.markdown]: ok, I removed the Oxford comma, and rephrased the second sentence to something that works better, I believe.

**Review by frenck — APPROVED** [review]
Thanks, @OnFreund!

I've tagged it for 0.108 and will be cherry-picked on the release of 0.108.

**Comment by OnFreund** [comment]
@frenck perfect, thanks!

**Comment by OnFreund** [comment]
@frenck BTW it probably makes sense to add Monoprice to the section titled "Many integrations now available to set up from the UI" in the upcoming release announcement.

**Comment by johanjongsma** [comment]
Hi, I have three amplifiers, stacked. I cannot fill zones 21 - 26 and 31-36.

Thanks, Johan

**Comment by OnFreund** [comment]
@johanjongsma the extension zones are disabled by default. Make sure you have "show hidden entities" turned on, and you'll be able to enable them.

**Comment by johanjongsma** [comment]
You're right. I completely misread it; it clearly said 'sources', where I treated as zones. Where would you  add the name of each zone? Through entities?

**Comment by OnFreund** [comment]
Yes, zones are managed through the entity registry. You can also rename the device which would automatically rename the entities.

**Comment by johanjongsma** [comment]
more feedback: I had a source name, but then I removed it. The old name stays. I had to enter a [space] to get rid of it.
and I still have issues with the zones, I can only use zones 11-16, please see the screenshots.

Thanks, Johan
image
image

**Comment by OnFreund** [comment]
It's all in the docs:

> In order to remove an existing source, you'll need to replace it with a space (simply removing its name will still keep it in place)

Unfortunately it's something that is at the HA level, not a specific integration, but I opened a ticket for it (https://github.com/home-assistant/frontend/issues/5337).

As for the missing zones - did you enable the entities?

**Comment by johanjongsma** [comment]
I dont know WHERE to enable them. A google search did not point me in the right direction. Can you?

**Comment by OnFreund** [comment]
In the entities tab, click the icon to the right of the search, and check "Show disabled entities". You can then open each entity to enable or disable it.

**Comment by johanjongsma** [comment]
Thanks, I just did; the zones are still not showing up in Lovelace.

**Comment by OnFreund** [comment]
You'll have to be a bit more specific - did you also enable them? Where are you not seeing them?

**Comment by frenck** [comment]
Guys, closed PR's and issues are not for handling support.

Please use Discord or the community forums for those things. This causes notifications with maintainers.

**Pull request merged**